micropub.php

```diff
@@ -432,18 +452,21 @@ private static function postprocess($post_id) {
       $geo = explode(':', $geo);
       $geo = explode(';', $geo[0]);
       $coords = explode(',', $geo[0]);
-      update_post_meta($post_id, 'geo_latitude', trim($coords[0]));
-      update_post_meta($post_id, 'geo_longitude', trim($coords[1]));
+      $args['meta_input']['geo_latitude'] = trim($coords[0]);
```

Comment:
wp_insert_post, which is used in new and updated post entries, takes anything stored in meta_input and updates the  post meta already. That is why I switched it. It has advantages and removes some code.

Also, checked...it is being stored.